x-pack/plugins/ml/common/types/capabilities.ts

```diff
@@ -106,6 +107,10 @@ export function getPluginPrivileges() {
         all: savedObjects,
         read: savedObjects,
       },
+      alerting: {
```

Comment:
@gmmorris Can you verify that these privileges are structured correctly?

Reply:
@jgowdyelastic could you please take a look as well?

Reply:
this looks correct to me